src/azul/tdr.py

```diff
@@ -142,6 +142,10 @@ def get_access_token(self) -> str:
 class TDRClient(SAMClient):
     """
     A client for the Broad Institute's Terra Data Repository code-named Jade.
+
+    Azul currently has no need to access the TDR API, but we expect it to become
+    necessary in the future. For now, this class is only used to verify that we
```

Comment:
The future has arrived. The PR that adds it is really close to landing: #2267. I think we can remove this paragraph.

Reply:
#2267 has been merged but `TDRClient` is still not used for any purpose other than access verification.

Reply:
The Azul service lambda, access TDR's DRS implementation and it needs to authenticate for that. This paragraph is incorrect and needs to be removed.